### PR TITLE
Implement contact form

### PR DESCRIPTION
**Pull request by Any97Cris**
Task is not complete. It remains to check the possibility of CRM.

### CODE REVIEW

**Comment by vitormattos** [comment]
Current status:

Form Before


Image

Form After


Image

**Comment by vitormattos** [comment]
Close #13

**Comment by giovannism20** [comment]
Thank u @vitormattos for the print, it's helpful for the code review.

@Any97Cris  About the style: your intention is to make style like the old form and we need to remember that the texts needs to be in english.

**Comment by vitormattos** [comment]
We have three possible ways for now:

1. Maintain as we can se now, using iframe. By this way will be necessary only do small changes at LimeSurvey side and create a specific template to LibreSign contact form
2. Implement the form direct by JS sending a post to LimeSurvey. I dislike this approach a bit because if we change the form from LimeSurvey side, will break the contact form
3. Pair with @DaianeAlvesRJ to create a form from SuiteCRM side and do the same what we made using LimeSurvey checking how to embed the generated form from SuiteCRM at point of contact form of this PR.

We need to think what will be more easy and best for now.

Using SuiteCRM would be good because the contact will turn to be a contact inside our CRM and sounds as a good idea.

**Comment by giovannism20** [comment]
I like the option 1 and 3

**Comment by vitormattos** [comment]
Form generated by SuiteCRM:

```html
<form id="WebToLeadForm" action="https://crm.librecode.coop/index.php?entryPoint=WebToPersonCapture" method="POST" name="WebToLeadForm">
<input name="last_name" id="last_name" type="text" required="" />
<input name="email1" id="email1" type="email" />
<input name="phone_mobile" id="phone_mobile" type="text" />
<input name="description" id="description" type="text" />
<input name="Submit" type="submit" value="Submit" />
<input name="campaign_id" id="campaign_id" type="hidden" value="91c4e277-95b8-ce28-91fd-658d9aa871ac" />
<input name="redirect_url" id="redirect_url" type="hidden" value="https://libresign.coop/thank-you-contact" />
<input name="assigned_user_id" id="assigned_user_id" type="hidden" value="89cce704-603d-79d7-b069-65660c3a0b7d" />
<input name="moduleDir" id="moduleDir" type="hidden" value="Contacts" />
</form>
```

**Review by vitormattos — APPROVED** [review]

File: source/thank-you-contact.blade.php [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+            currLink.classList.add("active");
+          } else {
+            currLink.classList.remove("active");
+          }
+        }
+      }
+
+      window.document.addEventListener("scroll", onScroll);
+    </script>
+  </body>
```
Comment (vitormattos) [inline source/thank-you-contact.blade.php]:
```suggestion
@endsection
```

File: source/thank-you-contact.blade.php [hunk 2]
```diff
@@ -0,0 +1,151 @@
+@extends('_layouts.main')
+
+@section('body')
+  <body>
```
Comment (vitormattos) [inline source/thank-you-contact.blade.php]: Move this to `_layouts.main`
```suggestion
```

**Review by vitormattos — CHANGES_REQUESTED** [review]

File: source/thank-you-contact.blade.php [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+                  <li class="nav-item">
+                    <a class="ud-menu-scroll" href="./html_pages/blog.html">Blog</a>
+                  </li>
+                </ul>
+              </div>
+            </nav>
+          </div>
+        </div>
+      </div>
+    </header>
```
Comment (vitormattos) [inline source/thank-you-contact.blade.php]: Move to `_layout.main`
```suggestion
```

File: source/thank-you-contact.blade.php [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+              .classList.remove("active");
+            currLink.classList.add("active");
+          } else {
+            currLink.classList.remove("active");
+          }
+        }
+      }
+
+      window.document.addEventListener("scroll", onScroll);
+    </script>
```
Comment (vitormattos) [inline source/thank-you-contact.blade.php]: Move to `_layout.main`
```suggestion
```

**Review by giovannism20 — APPROVED** [review]

**Review by vitormattos — APPROVED** [review]

**Pull request merged**